### PR TITLE
feat(VLR.gg): add presence

### PR DESCRIPTION
**Pull request by MadsonMendes**
**Describe the changes in this pull request:**
This time, actually adding the presence. First time doing this, sorry

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image
image
image
image
image
image
image
image
image
image

</details>

### CODE REVIEW

File: websites/V/VLR.gg/presence.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+let eventicon: HTMLImageElement;
+
+let playername: HTMLElement;
+
+presence.on("UpdateData", async () => {
+  title = document.querySelector("head > title");
+  rankinginfo = document.querySelector(".wf-tmp-h .normal");
+  teamicon = document.querySelector(".wf-avatar.team-header-logo img");
+  eventicon = document.querySelector(".wf-avatar.event-header-thumb img");
+  playername = document.querySelector(".player-real-name.ge-text-light");
```
Comment (EncryptedDev) [inline websites/V/VLR.gg/presence.ts:19]: Values of these never change, use constants instead

File: websites/V/VLR.gg/presence.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  const presenceData: PresenceData = {
+    largeImageKey: "vlr",
+    details: "Browsing page",
+    startTimestamp: browsingTimestamp,
+    state: title.textContent.replace(" | VLR.gg", ""),
+  };
+  /*Clear title in important pages*/
+  if (title.textContent.includes("Valorant esports coverage | VLR.gg")) {
+    presenceData.details = "On home";
+    presenceData.state = "";
```
Comment (EncryptedDev) [inline websites/V/VLR.gg/presence.ts:30]: Instad of using empty strings, use `delete`

File: websites/V/VLR.gg/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "service": "VLR.gg",
+  "altnames": ["vlr"],
+  "description": {
+    "en": "The major Valorant esports coverage website, featuring competitive news, schedule, match results, team rankings, stats, and forums.",
+    "pt": "O principal site de cobertura de esports do Valorant, com notícias competitivas, programação, resultados de partidas, classificações de equipes, estatísticas e fóruns."
+  },
+  "url": "www.vlr.gg",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/RVparK5.png",
```
Comment (EncryptedDev) [inline websites/V/VLR.gg/dist/metadata.json:15]: Logo must be 512z512

**Review by EncryptedDev — DISMISSED** [review]
Hi @MadsonMendes 👋
Thank you for contributing to PreMiD Presences for the first time. I have found a few things which you should take action for: 
- Unformatted code, please run `prettier --write .` 

If you need anything, drop me a message on Discord

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Take a look at my preview [review](https://github.com/PreMiD/Presences/pull/5855#pullrequestreview-890444750)

File: websites/V/VLR.gg/presence.ts (hunk 1 above)
Comment (MadsonMendes) [reply websites/V/VLR.gg/presence.ts:19]: Using consts I can't get an ".src" in line 67 and 76, should I change all of them to consts anyway?

**Comment by MadsonMendes** [comment]
Finally everything looks fine

File: websites/V/VLR.gg/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "Mad",
+    "id": "653742649798295560"
+  },
+  "service": "VLR.gg",
+  "altnames": ["vlr"],
```
Comment (Bas950) [inline websites/V/VLR.gg/dist/metadata.json:8]:
```suggestion
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Alt name can be removed since it's fully in the service name

File: websites/V/VLR.gg/presence.ts [hunk 5]
```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+		clientId: "939236497985536070"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let title: HTMLElement,
```
Comment (EncryptedDev) [inline websites/V/VLR.gg/presence.ts:5]: These variables never change, think about making these constants

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Use tabs to indent `metadata.json`

File: websites/V/VLR.gg/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.6",
+	"author": {
+		"name": "Mad",
+		"id": "653742649798295560"
+	},
+	"service": "VLR.gg",
+	"altnames": ["vlr"],
```
Comment (Slowlife01) [inline websites/V/VLR.gg/dist/metadata.json:8]:
```suggestion

```

File: websites/V/VLR.gg/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+	"description": {
+		"en": "The major Valorant esports coverage website, featuring competitive news, schedule, match results, team rankings, stats, and forums.",
+		"pt": "O principal site de cobertura de esports do Valorant, com notícias competitivas, programação, resultados de partidas, classificações de equipes, estatísticas e fóruns."
+	},
+	"url": "www.vlr.gg",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/RVparK5.png",
```
Comment (Slowlife01) [inline websites/V/VLR.gg/dist/metadata.json:15]: Must be at least 512x512

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/V/VLR.gg/presence.ts [hunk 8]
```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+		clientId: "939236497985536070"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	title: HTMLElement = document.querySelector("head > title"),
```
Comment (Bas950) [inline websites/V/VLR.gg/presence.ts:5]:
```suggestion
	title = document.querySelector<HTMLElement>("head > title"),
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
just type the selectors like that

**Pull request closed without merging**